spec/features/delete_a_photo_spec.rb

```diff
@@ -33,7 +33,7 @@
     click_link entry.title
     root_photo = "#{Rails.root}/spec/support/images/example_photo.png"
     attach_file "Entry Photo", root_photo
-    click_button 'Add Photo'
+    find("#add-photo",:visible=>false).click
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.